recirq/otoc/otoc_circuits.py

```diff
@@ -0,0 +1,571 @@
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions for generating OTOC circuits."""
+import itertools
+from typing import Sequence, Union, Tuple, List, Dict, Set, Any, Optional
+
+import cirq
+import numpy as np
+
+from recirq.otoc.parallel_xeb import build_xeb_circuits
+from recirq.otoc.utils import cz_to_sqrt_iswap
+
+_CORRECTION_FILE = Dict[Tuple[Tuple[int, int], Tuple[int, int]], cirq.Circuit]
```

Comment:
What is this? Why is it called a "file"? Please document here or in the argument docs for `build_otoc_circuits`

Reply:
It represents the two-qubit gates after parallel XEB correction for each qubit pair. It was called "file" because I usually save them... I changed the name to _GATE_CORRECTIONS to be more transparent and also documented this type.